src/utils.py

```diff
@@ -5369,6 +5369,17 @@ def subset_fonts(doc: pymupdf.Document, verbose: bool = False, fallback: bool =
     This method by default uses MuPDF's own internal feature to create subset
     fonts. As this is a new function, errors may still occur. In this case,
     please fall back to using the previous version by using "fallback=True".
+    Fallback mode requires the external package 'fontTools'.
+
+    Args:
+        fallback: use the older deprecated implementation.
+        verbose: only used by fallback mode.
+
+    Returns:
+        The new MuPDF-based code returns None.  The deprecated fallback
+        mode returns 0 if there are no fonts to subset.  Otherwise, it
+        returns the decrease in fontsize (the difference in fontsize),
+        measured in bytes.
```

Comment:
@JorjMcKie please check this "measured in bytes": it based only on reading these two lines:
```python
        new_fontsize += len(new_buffer)

    return old_fontsize - new_fontsize
```